src/app.js

```diff
@@ -1 +1,22 @@
+/* eslint-disable no-console */
 // write code here
+const fs = require('fs');
+const path = require('path');
+
+function moveFile() {
+  const [from, to] = process.argv.slice(2);
```

Comment:
Consider adding validation to check if both 'from' and 'to' arguments are provided. This will prevent potential runtime errors if the script is executed without the necessary parameters.